source/kotlin/src/test/kotlin/io/github/cosinekitty/astronomy/Tests.kt

```diff
@@ -5,12 +5,43 @@ import org.junit.jupiter.api.Test
 import org.junit.jupiter.params.ParameterizedTest
 import org.junit.jupiter.params.provider.CsvSource
 
-class Tests 
-{
+class Tests {
     @Test
-    fun `test DeltaT calculation`()
-    {
-        val time = AstroTime(0.0);
-        assertEquals(time.tt, 0.0007389709440951036);
+    fun `test deltaT calculation`() {
+        val time = AstroTime(0.0)
+        assertEquals(0.0007389709440951036, time.tt)
+    }
+
+    @ParameterizedTest
+    @CsvSource(
+        value = [
+            "2000, 1, 1, 12, 0, 0, 0, '2000-01-01 12:00:00.0 +0000'",
+            "2022, 1, 1, 12, 0, 0, 8036, '2022-01-01 12:00:00.0 +0000'",
+            "2022, 1, 1, 18, 0, 0, 8036.25, '2022-01-01 18:00:00.0 +0000'",
+        ]
```

Comment:
Just to note, there is also `@CsvFileSource` which can be used to share tests between the languages

Reply:
Meaning CsvFileSource accepts a file name and runs tests from it